WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -1,6 +1,9 @@
 import Foundation
 import WordPressFlux
 
+import class AutomatticTracks.CrashLogging
```

Comment:
Huh, I don't recall seeing us use this syntax anywhere else – I didn't even know it existed. Is there any advantage doing this over just importing `AutomatticTracks` and `Alamofire`?

Reply:
There are 2 advantages that I see:

1) It only makes the imported symbols available in the MediaCoordinator: I don't need this class to have access to the entirety of Alamofire, it just needs to know about AFError. Same with CrashLogging — I don't need the entirety of Tracks — all actual tracking needs should probably be handled by `WPAppAnalytics`
2) It indicates intent for the next person to read this code — nobody will ask "huh, why is this random class pulling in Alamofire for?" if it's clear that we just need that one specific object.


If you think the being unfamiliar syntax negates those, I'm happy to remove it :)

Reply:
Thanks for the explanation – I think it probably makes sense to keep it in that case.